packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity.jsx

```diff
@@ -18,6 +19,20 @@ const ProofOfIdentity = ({
     routeBackInApp,
     should_allow_authentication,
 }) => {
+    // next useEffect implements seo requirements
+    React.useEffect(() => {
+        const description_content = 'Submit your proof of identity documents to verify your account and start trading';
+        const title_content = 'Account Verification | Deriv app';
+
+        const restoreMetaTagWithOGDescription = changeMetaTagWithOG('description', description_content);
+        const restoreMetaTagWithOGTitle = changeMetaTagWithOG('title', title_content);
+
+        return () => {
+            restoreMetaTagWithOGDescription();
+            restoreMetaTagWithOGTitle();
```

Comment:
are these supposed to revert back the OG desc and title?

Reply:
yes, as we discussed in the channel. this behaviour has been agreed with Swati.
for other page she will provide other requirements later

Reply:
If that is the case then on unmounting the component, we need to revert back the title and desc to what they used to be. is that correct?

Reply:
yes, I believe so. That was I tried to implement here